framework/src/base/Moose.C

```diff
@@ -233,6 +235,8 @@ addActionTypes(Syntax & syntax)
   registerTask("check_integrity_early_physics", false);
   registerTask("setup_quadrature", true);
 
+  registerTask("mesh_modifiers", false);
```

Comment:
same

Reply:
I don't see a registerTask for userObjects